homeassistant/components/google_weather/quality_scale.yaml

```diff
@@ -0,0 +1,70 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
```

Comment:
If there are no actions, let's make it exempt

Reply:
I had the [weather.get_forecasts](https://www.home-assistant.io/integrations/weather/#action-weatherget_forecasts) in mind but that's from the platform so I guess it doesn't count. I'll make it exempt.